### DISCUSSION TITLE
Cannot see images in README

### DISCUSSION THREAD

**Discussion by yashs662** [community/community#40594] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I am experiencing slow load times for the README for any repo, my internet connection is stable and I am consistently getting 150-200 Mbps up and down. The images show the broken icon and when I click on the icon and go to the specific image file in the repo it does not load

**Comment by ossanna16** [comment]
Hi there @yashs662 and welcome to our community! Thank you for the feedback!

**Comment by yashs662** [comment, marked as answer]
Fixed, it! (I changed my DNS to Cloudflare and flushed the DNS cache and it's working now) was a problem on my end sorry!

**Reply by Prakash4844** [reply]
I, too, changed my DNS to Cloudflare and flushed DNS cache, but the issue still persists.

Edit: Actually it worked after restarting my PC.
